### PR TITLE
Completions update

### PR DESCRIPTION
**Pull request by 239**
## Description

- Added completions for `fd` and `windscribe`
- Updated previously added completions

## TODOs:

- [ ] Changes to fish usage are reflected in user documentation/manpages.
- [ ] Tests have been added for regressions fixed
- [x] User-visible changes noted in CHANGELOG.md

### CODE REVIEW

File: share/completions/fd.fish [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#fd 7.4.0
```
Comment (faho) [inline share/completions/fd.fish]: fd ships its own completions, please submit any enhancements to them upstream.

File: share/completions/cryptsetup.fish [hunk 2]
```diff
@@ -1,92 +1,93 @@
...
-complete -c cryptsetup -l use-random -d "Use /dev/random for generating volume key"
-complete -c cryptsetup -l use-urandom -d "Use /dev/urandom for generating volume key"
-complete -c cryptsetup -l uuid -d "UUID for device to use"
-complete -c cryptsetup -l veracrypt -d "Scan also for VeraCrypt compatible device"
-complete -c cryptsetup -l veracrypt-pim -d "Personal Iteration Multiplier for VeraCrypt compatible device"
-complete -c cryptsetup -l veracrypt-query-pim -d "Query Personal Iteration Multiplier for VeraCrypt compatible device"
-complete -c cryptsetup -l verbose -s v -d "Shows more detailed error messages"
-complete -c cryptsetup -l verify-passphrase -s y -d "Verifies the passphrase by asking for it twice"
-complete -c cryptsetup -l version -s V -d "Print package version"
+complete -c cryptsetup -l active-name                        -d "Override device autodetection of dm device to be reencrypted"
```
Comment (faho) [inline share/completions/cryptsetup.fish]: Please stop with the alignment - it's not worth it, and will be clobbered by `fish_indent`, our autoformatter.

See build_tools/style.fish.

File: share/completions/keybase.fish [hunk 3]
```diff
@@ -40,87 +41,81 @@ set -l keybase_fs_sync disable enable h help show
...
-complete -c keybase -f -n "$ends keybase" -l tor-mode
-complete -c keybase -f -n "$ends keybase" -l tor-proxy
-complete -c keybase -f -n "$ends keybase" -l updater-config-file
-complete -c keybase -f -n "$ends keybase" -l use-default-log-file
-complete -c keybase -f -n "$ends keybase" -l use-root-config-file
-complete -c keybase -f -n "$ends keybase" -l user-cache-size
-complete -c keybase -f -n "$ends keybase" -l vdebug
-complete -c keybase -f -n "$ends keybase" -l version
-complete -c keybase -f -n "$ends keybase" -s v -d --version
+complete -c keybase -f -n "$ends keybase" -l api-dump-unsafe                   -d ""
```
Comment (faho) [inline share/completions/keybase.fish]: Please don't add useless empty descriptions.

File: share/completions/zpaq.fish [hunk 4]
```diff
@@ -9,36 +9,36 @@ set -l command3 x extract
 
 #commands
 complete -c zpaq -f -n "not $seen $commands" -a "$commands"
-complete -c zpaq -f -n "not $seen $commands" -a a -d add
-complete -c zpaq -f -n "not $seen $commands" -a add -d "Append files to archive if dates have changed"
-complete -c zpaq -f -n "not $seen $commands" -a x -d extract
+complete -c zpaq -f -n "not $seen $commands" -a a       -d "add"
```
Comment (faho) [inline share/completions/zpaq.fish]: These quotes were removed on purpose, by `fish_indent`.

File: share/completions/fd.fish (hunk 1 above)
Comment (239) [reply share/completions/fd.fish]: You are right, I forgot about `fish_update_completions` on one host.

File: share/completions/cryptsetup.fish (hunk 2 above)
Comment (239) [reply share/completions/cryptsetup.fish]: Sorry, I align it only on my side for better overview and forgot about `fish_indent` before pushing.

File: share/completions/keybase.fish (hunk 3 above)
Comment (239) [reply share/completions/keybase.fish]: Keybase completions are still WIP and not completed. Should I remove the empty placeholders for now?

Comment (faho) [reply share/completions/keybase.fish]: If you're still working on them, leave it.

**Comment by faho** [comment]
Merged, sorry for the delay!

**Pull request merged**